### PR TITLE
[RED-659] Add Aviary.yaml

### PR DESCRIPTION
**Pull request by infosec-wf**
This is an automated pull request to migrate existing Raven config files to the new aviary.yaml file in individual repos. This PR requires infosec approval prior to merging. Please do not close this PR or merge it prior to this approval.

Please reach out in the "Infosec Forum" HipChat room with any questions.

@infosec-wf, @jayudey-wf

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
Number of Findings: 1
* Watched file ```aviary.yaml``` added

As a result of the finding(s) listed above, a security review is required for this pull request.

File: aviary.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+version: 1
+
+exclude:
+  - tests?/
```
Comment (evanweible-wf) [inline aviary.yaml]: @infosec-wf Could we also exclude all `.md` files? We mention a few things like `x-xsrf-token` that get picked up by raven.

Comment (jeffreymelvin-wf) [inline aviary.yaml]: sorry for the delay on responding to this, we are working through around 160

**Comment by evanweible-wf** [comment]
+1

**Comment by evanweible-wf** [comment]
@infosec-wf @jeffreymelvin-wf could we get this PR rebased against the latest w_transport master? That should fix the failing CI build

**Comment by matthewlorimor-wf** [comment]
Security +1

**Comment by evanweible-wf** [comment]
+1 @jayudey-wf ready for you

**Comment by matthewlorimor-wf** [comment]
+10 on the config

**Comment by jayudey-wf** [comment]
QA Resource Approval: +10

- [ ] Testing instruction
- [ ] Dev +1's
- [ ] Dev/QA +10 with detail of what was tested
 - performed and documented by Matthew Lorimor
- [ ] Unit test created/updated
- [ ] All unit tests pass

Merging into master.

**Pull request merged**